tests/integration/test_kube_orchestrator.py

```diff
@@ -90,7 +89,7 @@ def _prepare(self) -> None:
         self._record.internal_hostname = f"{self.id}.{namespace}"
         if self.is_named:
             self._record.internal_hostname_named = (
-                f"{self.name}{JOB_NAME_SEPARATOR}{self.project_name}.{namespace}"
+                f"{self.host_segment_named}.{namespace}"
```

Comment:
I think we could remove `_prepare` method and use [init_job_internal_hostnames](https://github.com/neuro-inc/platform-api/blob/master/platform_api/orchestrator/job.py#L880) instead